### PR TITLE
Edited some typos

### PR DESCRIPTION
**Pull request by torimcc**

### CODE REVIEW

File: client-tutorial/README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
...ny other docker-compose you can edit the launch config for the service (such as adding other tooling you might want such as MySQL or elastic search **(note using the containerized versions of a database may not be storage backed, as docker by default does not provide system storage to a container)**. Other dependecies you will need are located in python itself such as Kivy (this application uses Kivy to generate a cross-platform GUI). Obviously that is not required and feel free to modify this demo to use your favorite front end (or turn it into a web service and use react or whatever). Kivy was just easy since the project will run as a local desktop application. Given the mysql dependency I chose to use PyMysql and SqlAlchemy (ORM) to handle the database access. If you hate ORMs, you are probably a *psychopath*, but it's a free country. You will also need httpx, the pydantic fhir.resources, and a few other pip includes. You don't need to manually add these if you are not running your application locally (i.e. in an IDE). In PyCharm you can click on the **python packages** button at the bottom and search for whatever package you need and it will automagially add them to your venv).
```
Comment (ccorley) [inline client-tutorial/README.md:9]: I just noticed "Linux For Health" as well.  Should we change that to LinuxForHealth?

Comment (dixonwhitmire) [reply client-tutorial/README.md:9]: Good catch @ccorley - yes LinuxForHealth is our current "branding" so to speak.

**Review by dixonwhitmire — CHANGES_REQUESTED** [review]
Let's update "Linux For Health" to "LinuxForHealth", and then we will be able to merge. Thank you so much for reviewing the documentation!

**Comment by torimcc** [comment]
Made that change to LinuxForHealth!

**Review by torimcc — COMMENTED** [review]
Made the changes

**Pull request merged**